src/backend/executor/nodeNestloop.c

```diff
@@ -684,9 +684,10 @@ splitJoinQualExpr(List *joinqual, List **inner_join_keys_p, List **outer_join_ke
 				}
 				break;
 
+			case T_DistinctExpr:
 			case T_Const:
 				/*
-				 * Constant expressions do not need to be splitted into left and
+				 * Distinct and constant expressions do not need to be splitted into left and
```

Comment:
This line is too long. I suppose `pgindent` should be run on this file (or this line should be shortened).